cla_frontend/assets-src/javascripts/app/partials/call_centre/case_detail.assign.html

```diff
@@ -105,7 +105,7 @@ <h2 class="ContactBlock-heading">{{ selected_provider.name }}</h2>
   </div>
 
   <div class="FormActions" ng-if="::canAssign()">
-    <button type="submit" name="assign-provider" class="Button" ng-disabled="!selected_provider.id">Assign provider</button>
+    <button type="submit" name="assign-provider" class="Button" ng-disabled="!selected_provider.id || assigning_provider_in_progress">Assign provider</button>
```

Comment:
Hey Said, just wanted to ask if you know anything about the `selected.provider.id`. Before your implementation, it never disabled right? So `selected.provider.id` doesn't do anything atm

Reply:
`!selected_provider.id` disables the button if there is more than one provider available and none is selected